python/yugabyte_db_thirdparty/library_checking.py

```diff
@@ -317,6 +319,9 @@ def __init__(self) -> None:
             "^.* => /lib/",
             "^.* => /usr/lib/x86_64-linux-gnu/",
             "^.* => /opt/yb-build/brew/linuxbrew",
+            "^.* => /nix/store/",
+            "/bin/sh:",
```

Comment:
It would be nice to understand where this pattern comes from.

Reply:
/nix/store is where everything in nixos lives including libraries, the `/bin/sh:` pattern is to exclude warnings generated by the tooling attempting to use /bin/sh which was doing something weird on nixos IIRC